doc/src_usermanual/nlpsparse.tex

```diff
@@ -104,7 +104,6 @@ \subsubsection{Specifying the optimization problem}
 \begin{lstlisting}
 bool eval_Jac_cons(const long long& n, const long long& m, 
 			          const double* x, bool new_x,
-			          const long long& nsparse, const long long& ndense, 
```

Comment:
Why was this line removed? Was this a copy-paste error?

Reply:
was copy-paste error indeed